src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -471,8 +470,13 @@ private void enqueueAndHandleVariant(final VariantContext variant, final Referen
             for ( final FeatureInput<? extends Feature> featureInput : manualLocatableFeatureInputs ) {
                 @SuppressWarnings("unchecked")
                 final List<Feature> featureList = (List<Feature>)featureContext.getValues(featureInput, hg19Interval);
+
+                // TODO: This is a little sloppy, since it checks every datasource twice.  Once for hg19 contig names and once for b37 contig names.
+                featureList.addAll(featureContext.getValues(featureInput));
```

Comment:
Can we not detect the reference and only add in the features with the correct names?

Reply:
I am only going to file an issue for this.

https://github.com/broadinstitute/gatk/issues/4798

Done.